server/routes/spotify.js

```diff
@@ -119,4 +163,38 @@ router.get('/refresh_token', (req, res) => {
     })
 })
 
+router.get('/tokens', jwt.verifyJWT, async (req, res, next) => {
+  try {
+    const userId = req.userId
+    const spotifyId = Spotify.locals.spotifyId
+    const tokens = await SpotifyToken.findOne({ userId, spotifyId })
```

Comment:
Nit token singular since you're calling findOne.

Reply:
I think I made it plural b/c the object returned has an access and refresh token.

Reply:
True, but it represents a single object in the database.